docs/src/capi/howto/external-etcd.md

```diff
@@ -0,0 +1,124 @@
+# Use external etcd with Cluster API
+
+To replace the built-in datastore with an external etcd to
+manage the Kubernetes state in the Cluster API (CAPI) workload cluster follow
+this `how-to guide`. This example shows how to create a 3-node workload cluster
+with an external etcd. 
+
+## Prerequisites
+
+To follow this guide, you will need:
+
+- [Clusterctl][clusterctl] installed
+- A CAPI management cluster initialised with the infrastructure, bootstrap and
+  control plane providers of your choice. Please refer to the
+  [getting-started guide][getting-started] for instructions.
+- Secured 3-node etcd deployment
+
+## Create Kubernetes secrets
+
+Create three Kubernetes secrets:
+
+- peaches-etcd-servers
+- peaches-etcd
+- peaches-apiserver-etcd-client
+
+```{note}
+Replace `peaches` with the name of your cluster.
+It is important to follow this naming convention for the secrets since the providers will be looking for these names.
+```
+
+Create the secret for the etcd servers:
+
+```
+kubectl apply -f - <<EOF 
+apiVersion: v1
+kind: Secret
+metadata:
+  name: peaches-etcd-servers
+  namespace: default
+
```

Comment:
keep the namespace out of the manifest. the namespace must match the namespace where the cluster is deployed in